_docs/style/colors.md

```diff
@@ -60,32 +107,144 @@ illustrations. They should never overpower the primary colors.
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#d8d8d8" color="#424242" csscode="Gray 900 on Gray 400" hex="#424242 on #D8D8D8" ratio="7.1 : 1" %}
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#BDBDBD" color="#424242" csscode="Gray 900 on Gray 500" hex="#424242 on #BDBDBD" ratio="5.4 : 1" %}
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#9E9E9E" color="#333333" csscode="Gray 950 on Gray 600" hex="#333333 on #9E9E9E" ratio="4.7 : 1" %}
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#757575" color="#000000" csscode="Gray 1000 on Gray 700" hex="#000000 on #757575" ratio="4.6 : 1" %}
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#6B6B6B" color="#EEEEEE" csscode="Gray 100 on Gray 750" hex="#EEEEEE on #6B6B6B" ratio="4.6 : 1" %}
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#616161" color="#e0e0e0" csscode="Gray 300 on Gray 800" hex="#E0E0E0 on #616161" ratio="4.7 : 1" %}
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#424242" color="#bdbdbd" csscode="Gray 500 on Gray 900" hex="#BDBDBD on #424242" ratio="5.4 : 1" %}
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#333333" color="#9e9e9e" csscode="Gray 600 on Gray 950" hex="#9E9E9E on #333333" ratio="4.7 : 1" %}
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#212121" color="#9e9e9e" csscode="Gray 600 on Gray 975" hex="#9E9E9E on #212121" ratio="6.0 : 1" %}
-  {% include chips.html class="hxCol hxSpan-12-xs hxSpan-3-md color-chip" bg-color="#000000" color="#757575" csscode="Gray 700 on Gray 1000" hex="#757575 on #000000" ratio="4.6 : 1" %}
+  {%for color_hash in site.data.colors-interactive-states%}
+    {%assign color4=color_hash[1]%}
+    <div class="hxCol hxSpan-12-xs hxSpan-4-sm hxSpan-2-md">
+    <h4>{{color4.name}}</h4>
+      {%for value_hash in color4.colors%}
+      {%assign value4=value_hash[1]%}
+      <div class="lesser-swatch" style="background-color:{{value4.value}}">
+        <div class="{{value4.type}}-type">
+          {{value4.number}}<br>
+          <span class="smaller-color-type">{{value4.value}}</span>
+        </div>
+      </div>
+      {%endfor%}
+    </div>
+  {%endfor%}
+
 </div>
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Application frame
+
+Use the steel and gray bands to construct the application frame.
+
+In general:
+
+- Use the steel band for the application header and menus.
+- Use gray 25 for body background.
+- Use gray 0 for white elements.
+- Use gray 300 for disabled text.
```

Comment:
this one surprised me. why are we calling out gray 300 in the app frame?

Reply:
@palindromebart , that came from @araiford . I'm not sure, tbh. Let's talk!